Sources/PackageModel/Product.swift

```diff
@@ -51,9 +54,14 @@ public class Product: Codable {
         }
         self.name = name
         self.type = type
+        self.ID = package.description.lowercased() + "_" + name
         self._targets = .init(wrappedValue: targets)
         self.testManifest = testManifest
     }
+
+    public func useIDAsName() {
```

Comment:
this is okay for now, but we probably want to add a FIXME here to explore a non-mutating option of achieve this